frontend/model/state.js

```diff
@@ -364,12 +353,13 @@ const getters = {
   },
   ourPayments (state, getters) {
     // Payments relative to the current month only
+    const monthlyPayments = getters.groupMonthlyPayments
+    if (!monthlyPayments || Object.keys(monthlyPayments).length === 0) return
```

Comment:
The UI was breaking because an exception was thrown, so I returned `undefined` here to fix it. The issue with returning an empty object is that it's not clear to me personally whether or not `{}` is a falsey value. Even if it is, that requires some deep JS knowledge to know. And other getters might rely on the return value of this to perform additional computations. This way they can also check if this getter returned a false value and return immediately without throwing an exception that breaks UI rendering.

Reply:
Alright, let's leave it as is for now. There might be something to improve in the calling UI code and getters, but it can be fixed in a separate issue.